dusty/commands/repos.py

```diff
@@ -69,9 +81,12 @@ def update_managed_repos(force=False):
     specs_repo = get_specs_repo()
     if not specs_repo.is_overridden:
         log_to_client('Updating managed copy of specs-repo before loading specs')
+        add_known_hosts_for_repos([specs_repo])
```

Comment:
Not sure I agree... there's also the cli `dusty repos update`.  Also we need to do it in two steps since we don't know the hosts in the specs until we pull the specs repo